src/app/utils/badge-color.ts

```diff
@@ -0,0 +1,26 @@
+import { getTheme } from 'office-ui-fabric-react';
+
+export function getMethodStyle (method: string) {
+  const currentTheme = getTheme();
+
+  method = method.toString().toUpperCase();
+  switch (method) {
+    case 'GET':
+      return currentTheme.palette.blue;
+
+    case 'POST':
+      return currentTheme.palette.green;
+
+    case 'PUT':
+      return currentTheme.palette.magentaDark;
+
+    case 'PATCH':
+      return currentTheme.palette.yellowDark;
+
+    case 'DELETE':
+      return currentTheme.palette.redDark;
+
+    default:
```

Comment:
POST already uses the color green, why are we not using a different color for verbs not catered for by the switch statement?

Reply:
because we do not expect to get others not catered for. However, I can pick a cool colour to display.

Reply:
It will be cool to pick a cool color

Reply:
Please use the guide below for color codes for REST API verbs. 
![image](https://user-images.githubusercontent.com/4668452/67399564-1b744080-f5b5-11e9-83f6-d3311406fb2f.png)
@jobala @cwahome

Reply:
And what would be the color for PATCH? It is missing from the list.